docs/virtual_machines/disks_and_volumes.md

```diff
@@ -1403,68 +1403,217 @@ Test awesome shareable disks150+0 records in
 
-## File Systems
+## Sharing Directories with VMs
 
-Currently, only `virtIO-FS` file system is supported.
-### virtIO-FS
-
-`virtIO-FS` makes a Persistent Volume Claim visible in the `kubevirt` VM.
+`Virtiofs` allows to make visible external filesystems to `KubeVirt` VMs.
+`Virtiofs` is a shared file system that lets VMs access a directory tree on the host.
+Further details can be found at [Official Virtiofs Site](https://virtio-fs.gitlab.io/).
 
 > **Note:** The **ExperimentalVirtiofsSupport** feature gate
 > [must be enabled](../operations/activating_feature_gates.md#how-to-activate-a-feature-gate)
-> to use this volume. This feature is available starting with KubeVirt v0.36.0.
+> to share directories with VMs. This feature is available starting with KubeVirt v0.36.0.
+
+### Sharing Persistent Volume Claims
+#### Cluster Configuration
 
+We need to create a new VM definition including the `spec.devices.disk.filesystems.virtiofs` and a PVC.
 Example:
 
 ```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  name: testvmi-fs
+spec:
+  domain:
+    devices:
+      disks:
+        - disk:
+            bus: virtio
+          name: containerdisk
+        - disk:
+            bus: virtio
+          name: cloudinitdisk
+      filesystems:
+        - name: virtiofs-disk
+          virtiofs: {}
+    resources:
+      requests:
+        memory: 1024Mi
+  volumes:
+    - name: containerdisk
+      containerDisk:
+        image: quay.io/containerdisks/fedora:latest
+    - cloudInitNoCloud:
+        userData: |-
+          #cloud-config
+          password: fedora
+          chpasswd: { expire: False }
+      name: cloudinitdisk
+    - name: virtiofs-disk
+      persistentVolumeClaim:
+        claimName: mypvc
+```
+
+#### Configuration Inside the VM
+The following configuration can be done in using startup script. See [cloudInitNoCloud](#cloudinitnocloud) section for 
+more details. 
+However, we can do it manually by login inside the VM and mounting it. 
+Here are examples of how to mount it in a linux and windows VMs:
+
+- Linux Example
+
+```bash
+$ sudo mkdir -p /mnt/disks/virtio
+$ sudo mount -t virtiofs virtiofs-disk /mnt/disks/virtio
+```
+
+- Windows Example
+
+  See [this](https://virtio-fs.gitlab.io/howto-windows.html) guide for details on startup steps needed for Windows VMs.
+
+### Sharing Node Directories
+
+It is allowed using [hostpaths](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath).
+The following configuration example is shown for illustrative purposes.
+However, the [PVCs](#sharing-persistent-volume-claims) method is preferred since hostpath are generally discouraged for
+security reasons.
+
+
+#### Configuration Inside the Node
+
+We need to log in the node which we want to share the directory with the VMs, to create the shared directory (if it is not
```

Comment:
nit
```suggestion
We need to log into the node of which we want to share the directory with the VMs, to create the shared directory (if it is not
```

Reply:
This is quite a complicated sentence so I'd like to suggest the following, starting with the objective and followed by the steps needed:
"To share the directory with the VMs, we need to log in to the node, create the shared directory (if it does not already exist), and set the proper SELinux context label `container_file_t` to the shared directory."
Let me know if I misunderstand.

Reply:
You have understand it perfectly fine. Thanks for the rework :)